### ISSUE TITLE
techbooster.ymlの「事」を「こと」に修正するルールの改善案

### ISSUE THREAD

**Issue by munierujp** [prh/rules#21]
`techbooster.yml`内に、漢字の「事」をひらくために次のようなルールが定められています。

```yaml
- expected: こと
  pattern:  /(記|大|仕|返|無|食|見|議|当)?事(?!情|件|前|後|象|例|実|体|態|項|務|業|柄|故)/
```

正規表現で熟語にはマッチしないようにしようとされていますが、これでは定義されていない熟語（「悪事」や「事変」など）がマッチしてしまいます。
それらの熟語がマッチしないように漢字を追加してもいいのですが、これはホワイトリスト形式そのものの問題であると考えます。
そこで、いっそのことパターンを「前または後を漢字に挟まれていない『事』」という、より広いものに変えるのはどうでしょうか（正規表現でうまく表現できるのかはわかりませんが）。

**Comment by vvakame** [comment]
返信が遅くてすみません。
ルール全体として、ある程度false positiveを許容する(false negativeよりマシ)のが、校正ツールとしては正しいかと思っています。
なので、ホワイトリスト式を辞めるのは若干腰が重いところではあります。
ご自身の手元でよさそうなパターンを試していただいて、このパターンだとうまくいきそう…！というのがあると嬉しいです(凡例を考えてチェックしやすいため
